### ISSUE TITLE
Metamask stuck once account access request is approved

### ISSUE THREAD

**Issue by rokso** [MetaMask/metamask-extension#9049]
Web app needs to access user account information from Metamask and prompt user to authorize the access. Metamask stuck once access is approved. Started seeing this issue in new update. 
I would mention that doesn't always fail.

**Expected behavior**
Metamask window should hide and website should have access to account data.

**Screenshots**
![Screen Shot 2020-07-22 at 10 05 42 AM](https://user-images.githubusercontent.com/14338548/88193137-ed369680-cc02-11ea-97f4-a50157f1609e.png)

**Browser details (please complete the following information):**
 - OS: OS X
 - Browser: Chrome
 - MetaMask Version: 8.0.x

**Comment by Gudahtt** [comment]
Thanks for the report! This looks like a duplicate of #8973. This should be fixed in the next release (v8.0.5)
